packages/performance/src/performance.ts

```diff
@@ -11,13 +11,7 @@ import {
   referenceTime,
   hasGlobal,
 } from './utilities';
-import {
-  Event,
-  EventType,
-  LifecycleEvent,
-  TimeToFirstPaintEvent,
-  TimeToFirstContentfulPaintEvent,
```

Comment:
Can `TimeToFirstPaintEvent` and `TimeToFirstContentfulPaintEvent` be removed from the types file now? Or are there other references?